x-pack/plugins/ml/public/util/custom_url_utils.js

```diff
@@ -96,3 +100,24 @@ function buildKibanaUrl(urlConfig, record) {
     return tokenValue !== null ? tokenValue : match;
```

Comment:
i know this is old code, but this condition should be wrapped in parentheses. 
`return (tokenValue !== null) ? tokenValue : match;`